EosPayload/drivers/led_driver.py

```diff
@@ -0,0 +1,48 @@
+import Adafruit_BBIO.GPIO as GPIO
+import logging
+import time
+
+from EosLib.packet.definitions import Device
+from EosPayload.lib.driver_base import DriverBase
+
+
+class LEDDriver(DriverBase):
```

Comment:
i hate that we're spawning an entire process with 2 threads for this...but i guess it's fine for now.  I feel like this could be consolidated into the orcheostrator health functionality, it really isn't risky code

Reply:
It could, but then we'd have to add system checks to orchEOStrator because BBIO won't play nice on other systems, and for now this is sufficient as a check to verify that Eos has started. A better solution is definitely warranted before next launch though.

Reply:
If you want I could have it die once we're in flight? But in my test CPU utilization was sitting comfortably at about 50% and memory usage was acceptable, so I think a mostly-sleeping process is ok.

Reply:
I have some ideas for how we could handle the BBIO and other platform-depended deps (ie, do the import in a try/catch, then import a mock interface on ModuleNotFoundError)

I don't think it's critical to turn this off in-flight